pennylane/_qubit_device.py

```diff
@@ -214,6 +215,47 @@ def _permute_wires(self, observable):
         "Prod",
     }
 
+    measurement_map = defaultdict(lambda: "")  # e.g. {SampleMP: "sample"}
```

Comment:
Just to make sure: any value now queried returns an empty string, correct?

Reply:
Exactly!